### PR TITLE
Add interactivity

### PR DESCRIPTION
**Pull request by Theophileaseh**
Add delete selected

### CODE REVIEW

**Review by Kingstalux — COMMENTED** [review]
Hi @Theophileaseh , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f644550476b6e5269456e6965632f67697068792e676966

## Invalid Code Review Request 🛑 

You have submitted a project, that has failed linter checks. Kindly fix the linters before submitting for a review

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/modules/storage.js [hunk 1]
```diff
@@ -1,44 +1,52 @@
...
+// const listValue = document.querySelector('.single-list-input');
 
-let listArray = [];
+export let listArray = []; // eslint-disable-line
 
-const addList = () => {
+/* eslint-disable no-unused-vars */
+
+export const addList = () => {
   let listCode = '';
```
Comment (Dipeshtwis) [inline src/modules/storage.js]: - [ ] I would recommend removing the commented code to make the codebase cleaner.

File: src/modules/storage.js [hunk 2]
```diff
@@ -57,3 +65,55 @@ window.addEventListener('DOMContentLoaded', () => {
...
+
+    const inputValue = e.target.value
+    const {id} = parent.id;
+
+    listArray.forEach(list => {
+      if(id === list.index){
+        list.description = inputValue
+      }
+    })
+
```
Comment (Dipeshtwis) [inline src/modules/storage.js]: - [ ] I would recommend removing the commented code to make the codebase cleaner.

File: src/modules/storage.js [hunk 3]
```diff
@@ -57,3 +65,55 @@ window.addEventListener('DOMContentLoaded', () => {
...
+    const { checked } = e.target;
+    const text = e.target.parentNode.querySelector('.single-list-input');
+
+    if (checked) {
+      text.classList.add('line-through');
+    } else {
+      text.classList.remove('line-through');
+    }
+  }
+});
```
Comment (Dipeshtwis) [inline src/modules/storage.js]: - [ ] When I checked the task, the status is not changing in the local storage. Please fix it.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @Theophileaseh, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

- [ ] When I did `npm i` and `npm start`, I got the blank screen and can't able to do anything. In the README, please mention how we are going to run the project in the local.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N.A.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by addod19 — APPROVED** [review]
### Hi @Theophileaseh, 

Your project is complete! Nice job implementing the ocal storage functionality and removing all commented piece of code :smile:  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## NB:
- [ ] IT IS HIGHLY RECOMMENDED TO MAINTAIN  THE SAME PR FROM PREVIOUS CODE REVIEWERS SO THAT WE CAN MONITOR THE COMMENTS FROM PREVIOUS REVIEWERS EASILY

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**